tests/test_app/tests/upload_tests.py

```diff
@@ -93,7 +92,7 @@ def test_metadata(self):
             "Date",
         }
         assert res.metadata is not None
-        self.assertTrue(metadata_attrs.issubset(res.metadata.keys()))
+        assert metadata_attrs <= set(res.metadata.keys())
```

Comment:
We already use Pytest, so we can use actual `assert`, instead of the unittest-style assert methods. Relying on Pytest actually gives prettier output, so changing this helped me to debug the test failure.

I'm not going to bother rewriting all the tests to actually use Pytest syntax, but it would probably be an improvement.